### PR TITLE
feat: made loser and winner models

### PR DESCRIPTION
**Pull request by 999Vik**

### CODE REVIEW

**Review by Ram-Raghav-S — CHANGES_REQUESTED** [review]
Looks good except add goals_to_quarter. Refer to schema drawing.

**Comment by 999Vik** [comment]
> Looks good except add goals_to_quarter. Refer to schema drawing.

Goals_in_quarter has been added

File: requirements.txt [hunk 1]
```diff
@@ -1,49 +0,0 @@
...
-six==1.16.0
-sortedcontainers==2.4.0
-tabulate==0.8.10
-tenacity==8.1.0
-toml==0.10.2
-tomlkit==0.11.6
-typeguard==2.13.3
-urllib3==1.26.13
-Werkzeug==2.2.3
-wrapt==1.14.1
```
Comment (harshithl1777) [inline requirements.txt:49]: Your requirements.txt is empty. Not sure what's causing this but we can't merge this into main, it'll cause the dependencies to be deleted. Can you check to see if there's an issue with your `venv`? You shouldn't be deactivating it before you commit / push.

File: server/models/loser.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from peewee import AutoField, ForeignKeyField, IntegerField, CharField
+from server.models.tournament import Tournament
+from server.models.game import Game
+from server.models.team import Team
+from server.models.init import BaseModel, db
+
+
+class Loser(BaseModel):
+    class Meta:
+        table_name = "loser"
```
Comment (harshithl1777) [inline server/models/loser.py:10]: Table name should be plural.

File: server/models/winner.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from peewee import AutoField, ForeignKeyField, IntegerField, CharField
+from server.models.tournament import Tournament
+from server.models.game import Game
+from server.models.team import Team
+from server.models.init import BaseModel, db
+
+
+class Winner(BaseModel):
+    class Meta:
+        table_name = "winner"
```
Comment (harshithl1777) [inline server/models/winner.py:10]: Table name should be plural.

**Review by harshithl1777 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**